cpp/tests/sampling/rw_low_level_test.cu

```diff
@@ -782,3 +782,121 @@ TEST_F(RandomWalksPrimsTest, SimpleGraphRandomWalk)
 
   ASSERT_TRUE(test_all_paths);
 }
+
+TEST(RandomWalksSpecialCase, SingleRandomWalk)
```

Comment:
Is there a reason this can't be `rw_low_level_test.cpp`?  We've been trying to migrate to .cpp files where possible since they compile faster.

Reply:
Yes. `check_col_indices()` calls `thrust`.

Reply:
Might be worth considering migrating that to a utility file at some point.

Note we should probably use `thrust::count_if` in that function due to this
https://github.com/NVIDIA/thrust/issues/1016